app/views/distributions/leap-15.0.html.erb

```diff
@@ -93,6 +93,19 @@
               <a href="https://download.opensuse.org/distribution/leap/15.0/live/openSUSE-Leap-15.0-KDE-Live-x86_64-Current.iso.sha256" class="card-link"><%= _('Checksum') %></a>
             </div>
           </div><!-- /.card -->
+          <div class="card">
+            <div class="card-body">
+              <h4 class="card-title">
+                <span class="typcn typcn-download-outline"></span>
+                Rescue LiveCD (650MB)
+              </h4>
+              <h6 class="card-subtitle mb-2 text-muted"><%= _("For CD and USB stick") %></h6>
+              <a href="https://download.opensuse.org/distribution/leap/15.0/live/openSUSE-Leap-15.0-Rescue-CD-x86_64-Current.iso" class="card-link"><%= _('Direct Link') %></a>
+              <a href="https://download.opensuse.org/distribution/leap/15.0/live/openSUSE-Leap-15.0-Rescue-CD-x86_64-Current.iso.meta4" class="card-link"><%= _('Metalink') %></a>
+              <a href="https://download.opensuse.org/distribution/leap/15.0/live/openSUSE-Leap-15.0-Rescue-CD-x86_64-Current.iso?mirrorlist" class="card-link"><%= _('Pick Mirror') %></a>
```

Comment:
should we maybe add here `target="_blank"`? the other 3 links are downloading something, but that one takes you to a different page :thinking:

Reply:
@guoyunhe does that mean that you are again adding the `blank`? :thinking: